src/pydata_sphinx_theme/assets/styles/variables/_fonts.scss

```diff
@@ -32,12 +32,11 @@ html {
 
   // Font family
   // These are adapted from https://systemfontstack.com/ */
-  --pst-font-family-base-system: -apple-system, BlinkMacSystemFont, Segoe UI,
-    "Helvetica Neue", Arial, sans-serif, Apple Color Emoji, Segoe UI Emoji,
-    Segoe UI Symbol;
-  --pst-font-family-monospace-system: "SFMono-Regular", Menlo, Consolas, Monaco,
-    Liberation Mono, Lucida Console, monospace;
-
+  --pst-font-family-base-system: -apple-system, blinkmacsystemfont, segoe ui,
+    "Helvetica Neue", arial, sans-serif, apple color emoji, segoe ui emoji,
+    segoe ui symbol;
+  --pst-font-family-monospace-system: "SFMono-Regular", menlo, consolas, monaco,
+    liberation mono, lucida console, monospace;
```

Comment:
I *think* this will be OK, but I'm paranoid about preserving font name capitalization (legacy of many font-related LaTeX compilation errors). Is there an automated way we can we check this still works as designed?